test/qa/test_allowlist.py

```diff
@@ -38,83 +41,138 @@ def teardown_function(function):
     logging.log()
 
 
-# Tests for 3.8.2 hotfix. Allowlist should not create routes.
-# Issue 400
+@pytest.mark.parametrize("allowlist_alias", lib.ALLOWLIST_ALIAS)
 @pytest.mark.parametrize("subnet_addr", lib.SUBNETS)
-def test_allowlist_does_not_create_new_routes_when_adding_deleting_subnets(subnet_addr):
-    output_before_add = sh.ip.route.show.table(205)
-    sh.nordvpn.allowlist.add.subnet(subnet_addr)
-    output_after_add = sh.ip.route.show.table(205)
-    sh.nordvpn.allowlist.remove.subnet(subnet_addr)
-    output_after_delete = sh.ip.route.show.table(205)
+@pytest.mark.parametrize("tech,proto,obfuscated", lib.TECHNOLOGIES)
+@pytest.mark.flaky(reruns=2, reruns_delay=90)
```

Comment:
why is this failing sometimes, there is no `connect` made?

Reply:
Good point :smile: